pyproject.toml

```diff
@@ -9,7 +9,10 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.25.0"
-gunicorn "^20.0.4"
+gunicorn = "^20.0.4"
+pytest = "^6.2.2"
```

Comment:
We can declare pytest/selenium etc. as dev-dependencies, which would allow us to skip installing them on e.g. production machines
You can do that by either manually moving these lines below, or originally by adding them with `poetry add <package> --dev` or `poetry add <package> -D`